schema/deploy/shipping/views.sql

```diff
@@ -669,4 +669,85 @@ comment on view shipping.sample_with_best_available_encounter_data_v1 is
     'Version 1 of view of warehoused samples and their best available encounter date and site details important for metrics calculations';
 
 
+create or replace view shipping.incidence_model_observation_v4 as
+
+    select encounter.identifier as encounter,
+
+           to_char((encountered at time zone 'US/Pacific')::date, 'IYYY-"W"IW') as encountered_week,
+
+           site.details->>'type' as site_type,
+           site.details->>'category' as site_category,
```

Comment:
why did you add `site_category`?

Reply:
This removed an extra join in `metadata_for_augur_build` with `warehouse.site` that was used to get the `site_category`. I thought we might as well include it in the modeling view